<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fm_container</class>
 <widget class="QWidget" name="fm_container">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>244</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QToolButton" name="tb_container">
       <property name="text">
        <string>...</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Container</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="fr_container">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="gb_basic">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Basic parameters</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="margin">
          <number>3</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="le_name">
           <property name="toolTip">
            <string>identification of the container in schema</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Resource:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="cb_resource">
           <property name="toolTip">
            <string>name of the resource hosting container when manually set</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ch_advance">
        <property name="text">
         <string>Show Advanced parameters</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gb_advance">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Advanced parameters</string>
        </property>
        <layout class="QGridLayout" name="gridLayout1">
         <property name="margin">
          <number>3</number>
         </property>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>policy:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2" colspan="5">
          <widget class="QComboBox" name="cb_policy">
           <property name="editable">
            <bool>true</bool>
           </property>
           <property name="maxVisibleItems">
            <number>3</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="3">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>working dir:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3" colspan="4">
          <widget class="QLineEdit" name="le_workdir"/>
         </item>
         <item row="2" column="0" colspan="4">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>container name:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="5" colspan="2">
          <widget class="QLineEdit" name="le_contname">
           <property name="toolTip">
            <string>name of the container when instanciated at runtime</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>O.S.:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="6">
          <widget class="QLineEdit" name="le_os"/>
         </item>
         <item row="4" column="0" colspan="7">
          <widget class="QCheckBox" name="ch_mpi">
           <property name="text">
            <string>container supports MPI</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="4">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Parallel library:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="4" colspan="3">
          <widget class="QLineEdit" name="le_parallel"/>
         </item>
         <item row="6" column="0" colspan="6">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>mem mb:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="6">
          <widget class="QSpinBox" name="sb_mem">
           <property name="toolTip">
            <string>required memory (megaBytes)</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0" colspan="6">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>cpu clock:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="6">
          <widget class="QSpinBox" name="sb_cpu">
           <property name="toolTip">
            <string>required cpu clock frequency (MHz)</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0" colspan="6">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>nb nodes:</string>
           </property>
          </widget>
         </item>
         <item row="8" column="6">
          <widget class="QSpinBox" name="sb_nbNodes"/>
         </item>
         <item row="9" column="0" colspan="6">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>nb proc per node:</string>
           </property>
          </widget>
         </item>
         <item row="9" column="6">
          <widget class="QSpinBox" name="sb_procNode"/>
         </item>
         <item row="10" column="0" colspan="6">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>nb compo nodes:</string>
           </property>
          </widget>
         </item>
         <item row="10" column="6">
          <widget class="QSpinBox" name="sb_nbCompoNodes"/>
         </item>
         <item row="11" column="0" colspan="2">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>type:</string>
           </property>
          </widget>
         </item>
         <item row="11" column="2" colspan="5">
          <widget class="QComboBox" name="cb_type">
           <property name="toolTip">
            <string>container type</string>
           </property>
           <property name="maxVisibleItems">
            <number>2</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
